manpages/pihole.8

```diff
@@ -273,11 +278,15 @@ Some usage examples
     Whitelist/blacklist manipulation
 .br
 
-    \fBpihole -w iloveads.example.com\fR  Add "iloveads.example.com" to whitelist
+    \fBpihole -w iloveads.example.com\fR       Add "iloveads.example.com" to whitelist
+.br
+    \fBpihole -b -d noads.example.com\fR       Remove "noads.example.com" from blacklist
 .br
-    \fBpihole -b -d noads.example.com\fR  Remove "noads.example.com" from blacklist 
+    \fBpihole -wild "^example.*$"\fR           Add "^example.*$" as a regex pattern - would
```

Comment:
@Mcat12 is this description correct? I'd assume that `-wild` needs to be called with just a domain and should not take regex as input